canonical/lxd-ui
# Use vitest instead of react-scripts for tests

## Done

- remove react-scripts
- add vitest
- change script runner

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described [here](../HACKING.md#setting-up-for-development).
2. Perform the following QA steps:
    - ensure ci is executing ts spec tests and succeeding
